packages/personal-data-erasure/src/main.js

```diff
@@ -201,22 +248,26 @@ export default class PersonalDataErasure {
     }
   }
 
-  static buildDeleteRequests(result, builder) {
-    let deleteUri = builder
+  static buildDeleteRequest(
+    result: Object,
```

Comment:
We should expand this type

Reply:
Hmm well, I see that in the `_deleteOne` function the `buildDeleteRequest` is called with `result` being typed as `ClientResult`.